functions/vercel/api/get/[table]/[id].ts

```diff
@@ -0,0 +1,48 @@
+/**
+ *
+ * @method GET
+ * @query ID
+ * @return JSON
+ *
+ * @endpoint /get/[id]
+ */
+import 'isomorphic-fetch'
+import type { VercelRequest, VercelResponse } from '@vercel/node'
+import type { TableNames } from '../../../_lib/types'
+import { getXataClient } from '../../../_lib/xata.codegen'
+// import { XataRecord } from '@xata.io/client'
+
+export default async function handler(
+  request: VercelRequest,
+  response: VercelResponse
+) {
+  const table = request.query.table as TableNames
+  const id = request.query.id as string
```

Comment:
Should we add zod to the stack to avoid those `as …`? So we can show a simple example how to validate request

Reply:
I don't know, should we? 🤔
or maybe that's a second example?

Reply:
If you don't want to add a dependency a simple trick is to do:

```js
const id = String(request.query.id ?? '')
```

This way you are sure that `id` is always a string.

Reply:
Thanks @gimenete . Much better than an assertion for this small example! :)

@fabien0102 you good postponing zod usage to a second sample app?

Reply:
Good to me